mod_ci/controllers.py

```diff
@@ -555,6 +555,17 @@ def start_ci():
 
         elif event == "pull_request":
             # If it's a PR, run the tests
```

Comment:
Maybe move this comment to after your changes, that'll make more sense.

Reply:
One second.